storage/migrations/02_runtimes.up.sql

```diff
@@ -207,6 +207,17 @@ CREATE TABLE chain.evm_token_balance_analysis
 );
 CREATE INDEX ix_evm_token_balance_analysis_stale ON chain.evm_token_balance_analysis (runtime, token_address, account_address) WHERE last_download_round IS NULL OR last_mutate_round > last_download_round;
 
+CREATE TABLE chain.evm_contracts
+(
+  runtime runtime NOT NULL,
+  contract_address oasis_addr NOT NULL,
```

Comment:
Would it make sense to instead (or in addition to) store the ethereum version of the address here? I can't actually think of a use case where knowing the oasis version of an evm contract address would even be needed/useful - IMO, it might be just confusing.

I would suggest storing the `contract_address` similarly as we decided to store `evm_log_signature` in https://github.com/oasisprotocol/oasis-indexer/pull/423/files (raw bytes of the ethereum address in a `BYTEA` field)

Reply:
Ah so when we return the data via the API, this will be linked to the other `RuntimeAccount` data which includes the address preimage aka the base64-encoded eth address. 

For the purposes of indexer-internal accounting, I think we decided a while back to always use oasis-style addresses. It is somewhat clunky, but it does keep things consistent, especially if we ever support non-evm runtimes.

Reply:
Makes sense :+1: Thanks for clarifying.